docs/universal-gateway/examples/front-door-pattern.mdx

```diff
@@ -0,0 +1,97 @@
+---
+title: "Implement the 'Front Door' Pattern"
+description: By using Cloud Endpoint and internal Agent Endpoints together, you loosely double your gateway and services for extra flexibility.
+sidebar_label: "The Front Door Pattern"
+---
+
+import ReserveDomain from "./snippets/_reserve-domain.mdx";
+import CloudEndpoint from "./snippets/_cloud-endpoint.mdx";
+import TryOut from "./snippets/_try-out.mdx";
+import Back from "./snippets/_back-to-examples.mdx";
+
+The "front door" pattern involves using a Cloud Endpoint, the forward-internal Traffic Policy action, and an internal Agent Endpoint to route traffic from the public internet to your service(s).
```

Comment:
We should explain the actual pattern itself and why it is useful before introducing how it's done on ngrok.